jib-maven-plugin/README.md

```diff
@@ -280,11 +283,33 @@ In this configuration, the image:
       <port>1000</port>
       <port>2000-2003/udp</port>
     </ports>
+    <labels>
+      <key1>value1</key1>
+      <key2>value2</key2>
+    </labels>
     <format>OCI</format>
   </container>
 </configuration>
 ```
 
+In this configuration, the image:
```

Comment:
I think we can leave this example out since:

1. It's for an unrecommended use case.
1. It takes up a lot of room for just showing a single configuration.